hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/events/PrisonerMovementsEventService.kt

```diff
@@ -158,7 +158,6 @@ private fun Prisoner.isNewAdmission(previousPrisonerSnapshot: Prisoner?) =
 
 private fun Prisoner.isNewAdmissionDueToMoveBooking(previousPrisonerSnapshot: Prisoner?) =
   previousPrisonerSnapshot?.bookingId == null &&
-    this.bookingId != null &&
     this.status == "ACTIVE IN"
 
```

Comment:
bookingId can not be null given the status is ACTIVE IN - this was just hack to get the tests to pass